packages/indexer/src/utils/state-machine.helper.ts

```diff
@@ -36,6 +34,18 @@ export const RequestMetadata = Struct({
 	claimed: bool,
 })
 
+export const ResponseMetadata = Struct({
+	offchain: Struct({
+		leaf_index: u64,
+		pos: u64,
+	}),
+	fee: Struct({
+		payer: H256,
+		fee: u128,
+	}),
+	claimed: bool,
+})
```

Comment:
I think we already have a similar struct: `RequestMetadata` why not just use that